coverage/src/BamView-WebComponent.js

```diff
@@ -385,7 +424,36 @@ class BamViewChart extends HTMLElement {
         }
         return true;
     }
-    
+
+    handleBedfileClick(message, buttonElement) {
+        this.clearActiveButtons(); // Ensure only one button can be active
+        buttonElement.classList.add('active');
+
+        const event = new CustomEvent('default-bedfile-selected', {
+            detail: { message },
```

Comment:
Rather than emitting IDs, let's emit URLs to the bed files. That way it will be easier to implement functionality later to let the user provide a URL to a custom bed, if they have one on S3 or something that they want to use. For the built-in ones you can just use placeholder paths like `/grch37.bed` and `/grch38.bed` for now.

Also let's change the events to be `bed-file-selected` `bed-file-removed`